src/transformers/convert_slow_tokenizer.py

```diff
@@ -1276,7 +1276,7 @@ def vocab(self, proto):
         return vocab
 
     def pre_tokenizer(self, replacement, add_prefix_space):
-        return None
+        return pre_tokenizers.Metaspace(replacement=replacement, prepend_scheme="never", split=False)
```

Comment:
actually we can probably just use a `pre_tokenizers.Split()` here, will try that

Reply:
TLDR we want to get the word ids